src/sys/inotify.rs

```diff
@@ -5,17 +5,17 @@
 //!
 //! Inotify is a Linux-only API to monitor filesystems events.
 //!
-//! For more documentation, please read [the inotify(7) man page.](http://man7.org/linux/man-pages/man7/inotify.7.html)
+//! For more documentation, please read [inotify(7).](http://man7.org/linux/man-pages/man7/inotify.7.html)
 //! 
 //! Example
 //! =======
 //!
 //! * Monitor all events happening in directory "test":
 //! ```rust,no_run
```

Comment:
Pro tip: rustdoc is smart enough to figure out that the syntax is Rust; you don't need to tell it.  It's ok if you want to leave that in there, but you don't need to.